### PR TITLE
Homework20

### PR DESCRIPTION
**Pull request by timurk8**

### CODE REVIEW

File: src/test/java/BaseTest.java [hunk 1]
```diff
@@ -131,31 +134,26 @@ protected static void addToPlaylist() {
...
             // Add to the first playlist
             WebElement playlistElement = playlistElements.get(0);
-            playlistElement.click();
+            wait.until(ExpectedConditions.elementToBeClickable(playlistElement)).click();
+            //playlistElement.click();
         }
     }
-        //Delete the first playlist if exists
+        //Delete the first playlist if exists (HW-19, HW-20)
         protected static void deleteFirstPlaylist()  {
```
Comment (Berichicko) [inline src/test/java/BaseTest.java:142]: If possible, avoid overly long methods. A long method is 20‒25 lines. If possible, you should split the method into several shorter ones or even create a new class for this.

**Comment by Berichicko** [comment]
Hello,

I have checked your homework 20, and I wanted to let you know that you're doing great! Keep up the excellent work!
Best wishes from the HW team (Alex, [aliaksandr@testpro.io](mailto:aliaksandr@testpro.io))!

804deb64dc406a20a75da05d91a1a69e

image

File: src/test/java/BaseTest.java (hunk 1 above)
Comment (timurk8) [reply src/test/java/BaseTest.java:142]: Thank you for your advice!

Comment (timurk8) [reply src/test/java/BaseTest.java:142]: Done
https://github.com/timurk8/A47-QA-Automation-Framework/pull/13

**Pull request merged**